src/Microsoft.ServiceFabric.Actors/Runtime/ActorStateProviderHelper.cs

```diff
@@ -496,6 +498,47 @@ internal static IActorStateProvider CreateDefaultStateProvider(ActorTypeInformat
 
         #region Private Helpers
 
+        private bool GetContinuationPointByActorCount<T>(
+            long previousActorCount,
+            IEnumerator<T> enumerator,
+            CancellationToken cancellationToken)
+        {
+            long currentActorCount = 0L;
+            bool enumHasMoreEntries = true;
+
+            // Skip the previous returned entries
+            while (currentActorCount < previousActorCount && enumHasMoreEntries)
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+
+                currentActorCount++;
+                enumHasMoreEntries = enumerator.MoveNext();
+            }
+
+            return enumHasMoreEntries;
+        }
+
+        private bool GetContinuationPointByActorStorageKey<T>(
+            string lastSeenActorStorageKey,
+            IEnumerator<T> enumerator,
+            Func<T, string> getStorageKeyFunc,
+            CancellationToken cancellationToken)
+        {
+            bool enumHasMoreEntries = true;
+            var storageKey = getStorageKeyFunc(enumerator.Current);
+
+            // Skip the previous returned entries
+            while (enumHasMoreEntries && string.Compare(storageKey, lastSeenActorStorageKey, StringComparison.OrdinalIgnoreCase) <= 0)
+            {
+                cancellationToken.ThrowIfCancellationRequested();
+
+                enumHasMoreEntries = enumerator.MoveNext();
+                storageKey = getStorageKeyFunc(enumerator.Current);

```

Comment:
What happens if there are no more entries? 
Consider a case where,
Items in store : Actor1... Actor10
LastSeenActorStorageKey: Actor10

Reply:
If Actor10 is the last actor and page size=10 then no continuation token will be sent. However, as discussed, this would cause an issue when page size=9 and Actor10 is deleted after the first page is fetched.

Updated the code with the necessary changes. Made the needed changes in RCActorStateProvider as well.